web/src/components/survey/SurveyCreatorForm.vue

```diff
@@ -70,7 +70,8 @@ export default class SurveyCreatorForm extends Vue {
     ace.edit("surveyCreatorContainer", {});
     const editorOptions = {
       isAutoSave: true,
-      showLogicTab: true
+      showLogicTab: true,
+      designerHeight: "766px"
```

Comment:
Designer height is still 766px, wont work for screens that are 1080p, 1440p, 2160p (4k)